android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/ContentJson.kt

```diff
@@ -95,6 +102,17 @@ class ContentJson(
                         content = reaction.get("content").asString,
                     )
                 )
+            } else if (obj.has("reactionV2")) {
+                val reaction = obj.get("reactionV2").asJsonObject
+                return ContentJson(
+                    ContentTypeReactionV2, FfiReaction(
+                        reference = reaction.get("reference").asString,
+                        action = getReactionV2Action(reaction.get("action").asString.lowercase()),
+                        schema = getReactionV2Schema(reaction.get("schema").asString.lowercase()),
+                        content = reaction.get("content").asString,
+                        referenceInboxId = ""
```

Comment:
Question: is that intentional left empty?

Reply:
yea that property is sort of stubbed in for now, as I noticed it was included in the xmtp-js reaction type: https://github.com/xmtp/xmtp-js/blob/80aa5ec4fb24b60def222537cebcb20c818ca993/content-types/content-type-reaction/src/Reaction.ts#L24

However xmtp-react-native has an existing ReactionContent type that didn't feel worth updating at this point since I'm not sure what referenceInboxId would actually be used for. 

definitely a code smell though, I added a clarifying comments here 👌: faad8de361aa2825e204bc87f04ae097973aab78